### DISCUSSION TITLE
Is it possible to connect to MySQL with SSL/TLS?

### DISCUSSION THREAD

**Discussion by Benjababe** [goravel/goravel#396] [Q&A]
Some MySQL instances such as Azure in my case comes with TLS/SSL enabled by default.

![image](https://github.com/goravel/goravel/assets/19561469/91878a94-87e8-4fc1-bd3f-7674ad24a9a1)
 
I was wondering if there is an option to pass a certificate/`.pem` file into the connection for the ORM. Without changing anything, I will get the following error as expected:

```bash
[Orm] failed to initialize database, got error Error 3159 (HY000): Connections using insecure transport are prohibited while --require_secure_transport=ON.
[Orm] Init mysql connection error: Error 3159 (HY000): Connections using insecure transport are prohibited while --require_secure_transport=ON.
```

I could set `require_secure_transport=off` in the MySQL database but I would rather not unless necessary. I've scoured through the [ORM pages](https://www.goravel.dev/ORM/getting-started.html) but couldn't find anything on it.

**Comment by hwbrzzl** [comment]
Good idea, I created a feature for this: https://github.com/goravel/goravel/issues/400
